tests/suites/test_suite_pk.data

```diff
@@ -13,135 +13,135 @@ depends_on:MBEDTLS_RSA_C:MBEDTLS_GENPRIME
 pk_utils:MBEDTLS_PK_RSA:MBEDTLS_RSA_GEN_KEY_MIN_BITS:MBEDTLS_RSA_GEN_KEY_MIN_BITS:(MBEDTLS_RSA_GEN_KEY_MIN_BITS /8):"RSA"
```

Comment:
There's one occurrence of `MBEDTLS_ECP_DP_SECP384R1_ENABLED` in `test_suite_psa_crypto.data` which should be replaced as well.